netctl/netctl.go

```diff
@@ -317,7 +311,11 @@ func listEndpointGroups(ctx *cli.Context) {
 		for _, group := range filtered {
 			policies := ""
 			if group["policies"] != nil {
-				policies = strings.Join(group["policies"].([]string), ",")
+				policyList := []string{}
+				for _, p := range group["policies"].([]interface{}) {
```

Comment:
yes, it was crashing with this message:
$ netctl group list 
Tenant  Group  Network  Policies
------  -----  -------  --------
panic: interface conversion: interface is []interface {}, not []string

goroutine 1 [running]:
github.com/contiv/netplugin/netctl.listEndpointGroups(0xc82008aea0)
	/opt/gopath/src/github.com/contiv/netplugin/netctl/netctl.go:318 +0x644
github.com/codegangsta/cli.Command.Run(0x7bb4c0, 0x4, 0x0, 0x0, 0x0, 0x0, 0x0, 0x7f2bd0, 0x14, 0x0, ...)
	/opt/gopath/src/github.com/contiv/netplugin/Godeps/_workspace/src/github.com/codegangsta/cli/command.go:131 +0x1055